daemon/src/common/FileSystem.cpp

```diff
@@ -2392,7 +2397,7 @@ std::vector<PakInfo> GetAvailableMapPaks()
 	std::vector<PakInfo> infos;
 	for ( const auto& pak : FS::GetAvailablePaks() )
 	{
-		if ( Str::IsPrefix("map-", pak.name) )
+		if (DAEMON_FS_LEGACY_PAKS || Str::IsPrefix("map-", pak.name))
```

Comment:
Why don't we use more high-level functions?
Like IsMapPak or something. Then it might be easier to change/extend the naming system